src/backbone.syphon.typeregistry.js

```diff
@@ -17,7 +17,11 @@ _.extend(TypeRegistry.prototype, {
   // found for the specified type, the default is
   // returned.
   get: function(type){
-    return this.registeredTypes[type] || this.registeredTypes['default'];
+    if (this.registeredTypes.hasOwnProperty(type)){
```

Comment:
It is a regular object instantiated in line 8, I can test on IE8 (I am on Windows anyway... *ducks*)